<?xml version="1.0" encoding="utf-8"?><testsuite errors="3" failures="0" name="pytest" skipped="0" tests="4" time="22.022"><testcase classname="test_case.test_crm_cust_manger.TestCustomerManagement" file="test_case/test_crm_cust_manger.py" line="156" name="test_split_customer_success[未分单学员-分单成功-四基-新招生-高分云信息单-13800138000-高分云辅导1-高分云辅导督导1-123456-保存成功]" time="2.654"></testcase><testcase classname="test_case.test_crm_cust_manger" file="test_case/test_crm_cust_manger.py" line="217" name="test_convert_student" time="0.001"><error message="test setup failure">file /Users/huishow/PycharmProjects/new_crm_poium/test_case/test_crm_cust_manger.py, line 218
  def test_convert_student(self, crm_url, browser1, counseling_supervision_account, pass_word, jigou_school_name,
E       fixture &apos;self&apos; not found
&gt;       available fixtures: adviser_account, adviser_account2, adviser_name, adviser_name2, browser1, browser_close, cache, capfd, capfdbinary, caplog, capsys, capsysbinary, cases_pathh, class_name, comm_content, counseling_supervision_account, counseling_supervision_name, course, crm_url, customer_birthday, date_today, doctest_namespace, education_account, education_name, educational_account, educational_name, human_resource_account, in_class_time, jigou_school_name, make_up_class, make_up_time, metadata, monkeypatch, pass_word, phone_number, pytestconfig, questions, record_property, record_testsuite_property, record_xml_attribute, recwarn, remarks, student_name, supervisor_account, tmp_path, tmp_path_factory, tmpdir, tmpdir_factory, today_date
&gt;       use &apos;pytest --fixtures [testpath]&apos; for help on them.

/Users/huishow/PycharmProjects/new_crm_poium/test_case/test_crm_cust_manger.py:218</error></testcase><testcase classname="test_case.test_crm_cust_manger" file="test_case/test_crm_cust_manger.py" line="243" name="test_create_account" time="0.001"><error message="test setup failure">file /Users/huishow/PycharmProjects/new_crm_poium/test_case/test_crm_cust_manger.py, line 244
  def test_create_account(self, browser1, pass_word, jigou_school_name, crm_url, counseling_supervision_account,
E       fixture &apos;self&apos; not found
&gt;       available fixtures: adviser_account, adviser_account2, adviser_name, adviser_name2, browser1, browser_close, cache, capfd, capfdbinary, caplog, capsys, capsysbinary, cases_pathh, class_name, comm_content, counseling_supervision_account, counseling_supervision_name, course, crm_url, customer_birthday, date_today, doctest_namespace, education_account, education_name, educational_account, educational_name, human_resource_account, in_class_time, jigou_school_name, make_up_class, make_up_time, metadata, monkeypatch, pass_word, phone_number, pytestconfig, questions, record_property, record_testsuite_property, record_xml_attribute, recwarn, remarks, student_name, supervisor_account, tmp_path, tmp_path_factory, tmpdir, tmpdir_factory, today_date
&gt;       use &apos;pytest --fixtures [testpath]&apos; for help on them.

/Users/huishow/PycharmProjects/new_crm_poium/test_case/test_crm_cust_manger.py:244</error></testcase><testcase classname="test_case.test_crm_cust_manger" file="test_case/test_crm_cust_manger.py" line="281" name="test_customer_recovery" time="0.001"><error message="test setup failure">file /Users/huishow/PycharmProjects/new_crm_poium/test_case/test_crm_cust_manger.py, line 282
  def test_customer_recovery(self, crm_url, browser1, counseling_supervision_account, pass_word, phone_number,
E       fixture &apos;self&apos; not found
&gt;       available fixtures: adviser_account, adviser_account2, adviser_name, adviser_name2, browser1, browser_close, cache, capfd, capfdbinary, caplog, capsys, capsysbinary, cases_pathh, class_name, comm_content, counseling_supervision_account, counseling_supervision_name, course, crm_url, customer_birthday, date_today, doctest_namespace, education_account, education_name, educational_account, educational_name, human_resource_account, in_class_time, jigou_school_name, make_up_class, make_up_time, metadata, monkeypatch, pass_word, phone_number, pytestconfig, questions, record_property, record_testsuite_property, record_xml_attribute, recwarn, remarks, student_name, supervisor_account, tmp_path, tmp_path_factory, tmpdir, tmpdir_factory, today_date
&gt;       use &apos;pytest --fixtures [testpath]&apos; for help on them.

/Users/huishow/PycharmProjects/new_crm_poium/test_case/test_crm_cust_manger.py:282</error></testcase></testsuite>